src/models/grant.js

```diff
@@ -58,11 +62,16 @@ export default (sequelize, DataTypes) => {
     stateCode: DataTypes.STRING,
     startDate: DataTypes.DATE,
     endDate: DataTypes.DATE,
+    inactivatedDate: DataTypes.DATE,
+    inactivatedReason: DataTypes.ENUM(inactivatedReasons),
     recipientId: {
       type: DataTypes.INTEGER,
       allowNull: false,
     },
     oldGrantId: DataTypes.INTEGER,
+    deleted: {
+      type: DataTypes.BOOLEAN,
```

Comment:
Did we give any thought to using the paranoid: true / soft delete functionality included with sequelize rather than spinning up a new column?

Reply:
I considered having `deletedAt `, but not specifically through `paranoid`. It looks like paranoid is a combination of deletion and scoping. Ideally, we would have scoping, which I originally implemented. However, with the scoping, even though all the test were passing, we were unable to save objectives on ARs with other entities. Apparently, `sequelize `didn't handle something correctly.

Reply:
`paranoid `might or might not have the same problem. We can tag it as a potential improvement if needed in the future (since this change involves a tight schedule with the users being notified in advance)